### ISSUE TITLE
ESP

### ISSUE THREAD

**Issue by emek1337** [Jire/Charlatano#301]
someone tested esp(skeleton, box) on faceit?

**Comment by AutisticFurry** [comment]
Yeah its a bit iffy but it generally works. not the best, sometimes people don't come up
